public/js/header-9.js

```diff
@@ -0,0 +1,14 @@
+let navToggle = document.querySelector(".nav__toggle");
+let navWrapper = document.querySelector(".nav__wrapper");
+
+navToggle.addEventListener("click", function () {
+    if (navWrapper.classList.contains("active")) {
+        this.setAttribute("aria-expanded", "false");
+        this.setAttribute("aria-label", "menu");
+        navWrapper.classList.remove("active");
+    } else {
+        navWrapper.classList.add("active");
+        this.setAttribute("aria-label", "close menu");
+        this.setAttribute("aria-expanded", "true");
+    }
+});
```

Comment:
Git likes to have an empty line at the bottom of a file. Let's add one

Reply:
Check